Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -489,6 +489,12 @@ Function Main {
                 $LISTarballUrlOld = $param.Replace("LIS_TARBALL_URL_OLD=","")
             }
         }
+        if( $LISTarballUrlCurrent -eq "LatestLIS" ) {
+            $LISTarballUrlCurrent=Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "echo ``curl -Ls -o /dev/null -w %{url_effective} http://aka.ms/lis``"
+        }
+        if( $LISTarballUrlOld -eq "LatestLIS" ) {
```

Comment:
how can the old LIS be the latest version?

Reply:
When Creating a new LIS RPM build,
LatestLIS will be LISTarballUrlOld and the generated LIS will be LISTarballUrlCurrent